CHANGELOG.md

```diff
@@ -6,6 +6,37 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### BREAKING CHANGES
+
+- **CUMULUS-1684**
```

Comment:
Not all of the contents of this list are breaking changes. The stuff that is just additions should be moved under the `Added` header, etc

Reply:
Fixed in fc0dbd4d7